ext/bcmath/libbcmath/src/recmul.c

```diff
@@ -38,23 +38,26 @@
 
 
 #if SIZEOF_SIZE_T >= 8
-#  define BC_MUL_UINT_DIGITS 8
-#  define BC_MUL_UINT_OVERFLOW (BC_UINT_T) 100000000
+#  define BC_VECTORIZED_MULTIPLICATION_VECTOR_SIZE 8
+/* The boundary number is computed from BC_VECTORIZED_MULTIPLICATION_VECTOR_SIZE times BASE */
```

Comment:
```suggestion
/* The boundary number is computed from BASE**BC_VECTORIZED_MULTIPLICATION_VECTOR_SIZE */
```

Reply:
Also, long name...

Reply:
Hmm... do you think `BC_MUL_VECTOR_SIZE` is too omission?

Reply:
Or how about changing `BC_INT_T`/`BC_UNIT_T` to `BC_SIGNED_VECTOR`/`BC_UNSIGNED_VECTOR` and defining this as `BC_VECTOR_SIZE` (or `BC_VECTOR_DIGITS`) in the header?

Reply:
I like the `BC_VECTOR` suggestion and I think the name `BC_VECTOR_BOUNDARY_NUM` is fine.

Reply:
Thx, since it gets complicated, I'll open a macro rename PR before this one and rebase when it gets merged.